services/QuillLMS/app/controllers/teachers/classroom_manager_controller.rb

```diff
@@ -275,6 +276,17 @@ def activity_feed
     render json: { data: TeacherActivityFeed.get(current_user.id) }
   end
 
+  private def format_classrooms_for_dashboard
+    classrooms = current_user.classrooms_i_teach
+
+    classrooms.map do |classroom|
+      classroom_obj = classroom.attributes
+      classroom_obj[:students] = classroom.students
+      classroom_obj[:student_count] = classroom.students.length
+      classroom_obj
```

Comment:
This is a syntax nit-pick, but whenever I see the end of a block just a variable declaration, I try to figure out another way to do things.  This could be:
```ruby
classroom.attributes.merge({ students: classroom.students, student_count: classroom.students.length })
```